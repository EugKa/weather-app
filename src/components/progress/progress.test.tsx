import React from 'react';
import { shallow } from "enzyme";
import { Progress } from "..";

const setUp = () => shallow(<Progress/>)

describe('Progress component', () => {    
    it("should render Progress component", () => {
        const component = setUp()
        expect(component).toMatchSnapshot()
    }) 
})