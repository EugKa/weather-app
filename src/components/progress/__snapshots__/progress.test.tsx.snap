// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Progress component should render Progress component 1`] = `
<ForwardRef(Box)
  className="main-pr"
  sx={
    Object {
      "display": "flex",
    }
  }
>
  <ForwardRef(CircularProgress) />
</ForwardRef(Box)>
`;
